lib/dstu2_resources.rb

```diff
@@ -305,6 +305,204 @@ module Resources
       ]
     }
 
+    DSTU2_CONDITION_BUNDLE ||= {
```

Comment:
+1. I changed most of the data inside the entry however, forgot to change it here. Will update the ids.